app/services/CheckoutService.scala

```diff
@@ -206,6 +207,7 @@ class CheckoutService(
           EitherT(salesforceService.repo.updateIdentityId(memberId, succ.userData.id.id)).swap.map(err =>
```

Comment:
Could pull succ.userData.id.id out into a val here (and perhaps rename succ whilst we're at it)